### PR TITLE
Changes for supporting Sortdb

### PR DESCRIPTION
**Pull request by masci**
### What does this PR do?

Continuation of #83 

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### CODE REVIEW

File: sortdb/datadog_checks/sortdb/check.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    def _get_sortdb_metrics(self, sortdb_url, metrics, instance_tags):
+        """
+        Get SortDB metrics
+        """
+        timeout = float(self.init_config.get('timeout', 10))
+        # Use a hash of the URL as an aggregation key
+        aggregation_key = md5(sortdb_url).hexdigest()
+
+        response = self._get_response_from_url(sortdb_url, timeout, aggregation_key, instance_tags)
+        print(response)
```
Comment (ofek) [inline sortdb/datadog_checks/sortdb/check.py]: Let's remove this

File: sortdb/metadata.csv [hunk 2]
```diff
@@ -0,0 +1,29 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration
+sortdb.stats.total_requests,GAUGE,,task,,Total number of requests,0,sortdb
+sortdb.stats.total_seeks,GAUGE,,task,,Total number of seeks,0,sortdb
+sortdb.stats.get_requests ,GAUGE,,task,,Total number of get requests,0,sortdb
```
Comment (nmuesch) [inline sortdb/metadata.csv]: There is an extra space on a few of these lines after the metric name. Not sure if the metrics metadata pipeline will play nicely, can we remove those?

File: sortdb/manifest.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "maintainer": "namrata.deshpande4@gmail.com",
+  "manifest_version": "1.0.0",
+  "name": "sortdb",
+  "short_description": "Datadog support for sortdb monitoring",
+  "guid": "806dcbd7-3686-4472-9435-2049729847c1",
+  "support": "contrib",
+  "supported_os": ["linux","mac_os","windows"],
```
Comment (nmuesch) [inline sortdb/manifest.json]: Can we add the fields like `create_events` so we can keep this more uniform with integrations-core checks?

File: sortdb/tests/test_sortdb.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    ("sortdb.stats.range_misses", 1),
+    ("sortdb.stats.range_requests.avg", 18),
+    ("sortdb.stats.range_requests.95percentile", 24),
+    ("sortdb.stats.range_requests.99percentile", 24),
+    ("sortdb.stats.db_size.bytes", 767557632),
+    ("sortdb.stats.db_mtime", 1435463934)
+)
+
+
+def test_check(aggregator, instance):
```
Comment (nmuesch) [inline sortdb/tests/test_sortdb.py]: 💜 💯

File: sortdb/tox.ini [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+envlist =
+    sortdb
+    flake8
+
+[testenv]
+platform = linux|darwin
+
+[testenv:sortdb]
+deps =
+    datadog_checks_base
```
Comment (ofek) [inline sortdb/tox.ini]: Let's remove this

**Comment by phrawzty** [comment]
I just want to mention how happy I am to see this moving forward. 😄

**Pull request merged**